stage2/04-install-umbrel/01-run.sh

```diff
@@ -22,12 +22,21 @@ EOF
 # Install Umbrel
 echo "Installing Umbrel..."
 echo
+if [ "${UMBREL_DEV}" == "0" ]; then
```

Comment:
Just wondering what the purpose of ```UMBREL_DEV``` is.. ?

Reply:
I don't know why I added that, I'll remove it.

Reply:
ok otherwise everything looks good